### PR TITLE
New banner design

### PR DESCRIPTION
**Pull request by martijnjanssen**
The slider was adding nothing to the webpage other than slower load times, we agreed to remove it.

The tabs at the bottom of the banner are clickable when there is an underline under the text, allowing the users to navigate quickly after reading the message.

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -119,7 +119,7 @@
       border-bottom: 2px solid var(--secondary-color);
     }
 
-    img {
+    .headerLogo {
```
Comment (martijnjanssen) [inline index.html]: This css property gave EVERY img item in the webpage a margin on the left, reaping havoc all over. I removed it 😛

Comment (Fastjur) [reply index.html]: Woopteedoo. Good catch

**Comment by martijnjanssen** [comment]
Huh, I thought I made it for smaller screens, at least 360px. I'll check.

**Comment by Fastjur** [comment]
I think the text I changed breaks it. The wrapping makes the text go above and below the yellow block

**Comment by martijnjanssen** [comment]
This is the updated banner right now, on a wide and a narrow screen:

screen shot 2018-01-23 at 14 04 37
screen shot 2018-01-23 at 14 05 05

**Pull request merged**